rcl/test/rcl/test_wait.cpp

```diff
@@ -265,11 +261,12 @@ TEST_F(CLASSNAME(WaitSetTestFixture, RMW_IMPLEMENTATION), multi_wait_set_threade
     size_t thread_id;
   };
   std::vector<TestSet> test_sets(number_of_threads);
+  // *INDENT-OFF* (prevent uncrustify from making unnecessary whitespace around [=])
   // Setup common function for waiting on the triggered guard conditions.
   auto wait_func_factory =
-    [count_target, retry_limit, wait_period](TestSet & test_set) {
+    [=](TestSet & test_set) {
```

Comment:
was this change required with this PR?

Reply:
Yes without this, I get warnings on macOS about unnecessary captures, but if I remove them and do not replace it with `=` then it fails to compile on Windows. Something called `Wunused-lambda-capture`, though it might be a bug in clang, see:

https://stackoverflow.com/questions/41960347/why-am-i-getting-an-unused-lambda-capture-warning

But this works just as well and produces no warnings or errors on any platform.